roles/ec2-s3-bucket/tasks/sub-create.yml

```diff
@@ -44,3 +44,25 @@
             --bucket {{ bucket_name }}  \
             --lifecycle-configuration file://lifecycle.json "
   when: "'lifecycle' in bucket_data and 'non_current_version_expiration' in bucket_data.lifecycle"
+
+- name: Get KMS Key metadata
+  command: "aws kms describe-key --key-id alias/{{ default_encryption_kms_keyname }}"
+  register: kms_metadata
+  check_mode: no
+  when: "'default_encryption_kms_keyname' in bucket_data"
```

Comment:
Check this condition carefully.
Do you need a default value? What happens if YAML Key default_encryption_kms_keyname is empty?

Reply:
The intention is that if default_encryption_kms_keyname is empty, this should be skipped, no encryption applied to the bucket